docs/sources/writing-guide/front-matter/index.md

```diff
@@ -101,15 +112,50 @@ labels:
     - enterprise
 ```
 
-For a **directory of pages** that describe a feature only available in Grafana Cloud, the branch bundle `_index.md` file front matter should include the following:
+#### Set cascading front matter
+
+When a directory of pages share the same front matter, it is convenient to set them all at once with cascading front matter.
+
+There are two forms of cascading front matter.
+The first is a `cascade` map of keys to values.
+The second is a sequence of mappings with the `_target` keyword.
+You cannot override the latter with the former.
+For more information about cascading front matter, refer to [Front Matter Cascade](https://gohugo.io/content-management/front-matter/#front-matter-cascade).
+
+{{% admonition type="note" %}}
+To avoid confusing behavior, set all cascading front matter with the sequence of mappings form.
+{{% /admonition %}}
```

Comment:
This doesn't seem like it should be a note—seems pretty central to what you're talking about here.

Reply:
+1